api/tests/provider_api_tests.py

```diff
@@ -52,3 +53,36 @@ def test_get_list_of_providers(self):
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         self.assertEqual(list(response.data[0].values())[1], self.provider1.person.name)
         self.assertEqual(len(response.data), 3)
+
+
+class ProviderFilteringTestCase(BaseAPITest):
```

Comment:
You can make the `BaseAPITestCase` really abstract and create a `ProviderAPITestCase` that handles all the tests.

The `BaseAPITest` is currently testing the provider api endpoint. We should aim to make it general in order to be able to use it for other APIs as well.